integrations/actix/src/lib.rs

```diff
@@ -287,10 +295,44 @@ pub fn handle_server_fns_with_context(
                                 }
                             }
                         }
-                        Err(e) => HttpResponse::InternalServerError().body(
-                            serde_json::to_string(&e)
-                                .unwrap_or_else(|_| e.to_string()),
-                        ),
+                        Err(e) => {
+                            let url = req
+                                .headers()
+                                .get(header::REFERER)
+                                .and_then(|value| {
+                                    Url::parse(
+                                        &Regex::new(
+                                            r"(?:\?|&)?server_fn_error=[^&]+",
+                                        )
+                                        .unwrap()
+                                        .replace(value.to_str().ok()?, ""),
+                                    )
+                                    .ok()
+                                });
+
+                            if let Some(mut url) = url {
+                                url.query_pairs_mut().append_pair(
+                                    "server_fn_error",
```

Comment:
Will need to use a key that matches specific server functions before merge.

Reply:
Yeah this should be fine, especially if `create_server_action` is responsible for checking its particular error in the URL, because it knows the server fn's unique name already. (Could use the ServerFnType::url()` or some form of it as the key)